sources/models/src/modeled_types/mod.rs

```diff
@@ -150,6 +151,20 @@ pub mod error {
             source: std::num::ParseIntError,
         },
     }
+
+    /// Creates a `ValidationError` with a consistent message for strings with regex validations
+    /// where the regex is too big to display to the user.
+    pub(crate) fn big_pattern_error<S1, S2>(thing: S1, input: S2) -> ValidationError
```

Comment:
No. But there are some very frequent patterns such as "regex validation error" and "big regex validation error". I anticipate figuring out some additional code re-use for the regex validations since most have the same validation code.